source/content.css

```diff
@@ -105,3 +105,23 @@
 .dropdown-menu-sw {
 	transform-origin: 90% top;
 }
+
+/* Restore "Delete branch" button in responsive branch list */
+@media (max-width: 1011px) {
+	.js-branch-row .Details-content--shown {
+		display: block !important;
+	}
+	form.js-branch-destroy {
```

Comment:
These two selectors can be merged.

Also, as a (probably) last change, this CSS could be moved to its own `always-show-branch-delete-buttons.css` file